### PR TITLE
upgrade to D9: push to origin; def copy pantheon.yml

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Closes Issue:** #6412 

## Summary

**[Migrate to D9](https://pantheon.io/docs/guides/drupal-9-migration/upgrade-to-d9)** -  It's safer to always copy `pantheon.yml` so that one exists rather than make the step conditional. Also fix the `git push` step to `origin` instead of `ic`.

## Effect

The following changes are already committed:

* 
*

## Remaining Work

The following changes still need to be completed:

- [x] 👍 
- [ ] pre-flight ✔️ 
- [ ] 🚀 

--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6420](https://pr-6420--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `59` | `3` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6420` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/27491/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/27491/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6420--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/partials/drupal-8-convert-to-composer.md [hunk 1]
```diff
@@ -43,23 +43,23 @@ This process involves significant changes to the codebase. We recommend you to d
...
-     git commit -m 'Copy my pantheon.yml'
-     ```
+  ```bash{promptUser:user}
+  git checkout master pantheon.yml
+  git add pantheon.yml
+  git commit -m 'Copy my pantheon.yml'
+  ```
 
-   If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
+ If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
```
Comment (greg-1-anderson) [inline source/partials/drupal-8-convert-to-composer.md:62]: We might more generally say that any value that is not needed from pantheon.yml can / should be removed. At a minimum, though, they should have a file and it should have `api_version: 1` in it.

**Review by greg-1-anderson — APPROVED** [review]
Minor optional advice wording, overall LGTM

File: source/partials/drupal-8-convert-to-composer.md (hunk 1 above)
Comment (EdwardAngert) [reply source/partials/drupal-8-convert-to-composer.md]: I modified [these lines to remind/reflect this](https://github.com/pantheon-systems/documentation/pull/6420/commits/8dee21155f7f3f1c2c8671d2b5052df633e44361#diff-14d96ef63a5b97fe59416ff30594a9b98d422d0fda45e712d0e2152c39451870R62-R64)
+ moving to Docs copy review

Comment (greg-1-anderson) [reply source/partials/drupal-8-convert-to-composer.md]: Looks great, thanks.

**Review by carolyn-shannon — APPROVED** [review]
Looks good. Future docs task for the convert to composer guide include renaming the title to Integrated Composer & adding context links.

**Pull request merged**